### ISSUE TITLE
Support alpine images

### ISSUE THREAD

**Issue by brandon-leapyear** [CircleCI-Public/aws-cli-orb#82]
We're running a CI Job in the `docker.bintray.io/jfrog/jfrog-cli-go:latest` image, and the orb doesn't install:
```
Installing AWS CLI v2
+ uname -a
+ grep Darwin
+ uname -a
+ grep 'x86_64 GNU/Linux'
+ uname -a
+ grep 'aarch64 GNU/Linux'
+ echo 'This platform appears to be unsupported.'
This platform appears to be unsupported.
+ uname -a
Linux 883adc3d22f7 4.15.0-1092-aws #98-Ubuntu SMP Wed Jan 6 22:22:51 UTC 2021 x86_64 Linux
+ exit 1

Exited with code exit status 1
```
It would be great if it could grep for `x86_64 Linux` in addition to `x86_64 GNU/Linux`.

Also, it seems like the orb doesn't work when run in `sh`. Maybe the orb can specify the shell to be `bash`?

**Comment by brivu** [comment]
Hey @brandon-leapyear, 

Thanks for your patience on this issue. Can you give me some more insight about your use case? Does Alpine provide any particular utility that our `cimg/base` does not?

Thank you again! 
Brian

**Comment by brandon-leapyear** [comment]
So this isn't relevant for us anymore because we removed the CI job that ran into this issue. But to respond to your question (and to solve the general problem it seems like 9 other people are having):

> We're running a CI Job in the `docker.bintray.io/jfrog/jfrog-cli-go:latest` image

We weren't using `cimg/base`, we were running a CI job that deploys files via the jfrog CLI, so we were using the jfrog image, and we needed to use the aws cli in the same job.

**Comment by LaurentGoderre** [comment]
This is also needed to install the CLI in Docker-in-Docker situation.

For example run:

```
docker run --rm docker uname -a
```

or

```
docker run --rm docker/compose uname -a
```

What you get is:

```
x86_64 Linux
```

**Comment by LaurentGoderre** [comment]
As a workaround, I move the authentication to it's own job then mount the `.aws` folder created to the other jobs

```yaml
authenticate:
    docker:
      - image: cimg/base:2022.03
    steps:
      - aws-ecr/ecr-login
      - run: |
          mkdir -p /tmp/workspace
          cp -r ${HOME}/.aws /tmp/workspace/.aws
      - persist_to_workspace:
          root: /tmp/workspace
          paths:
            - '.aws'
```

Then

```yaml
- attach_workspace:
          at: /tmp/workspace
      - run:
          name: Copy AWS Auth config
          command: |
            cp -r /tmp/workspace/.aws ${HOME}/.aws
```

**Comment by brivu** [comment]
This was resolved with PR #98
